src/app/content/studyGuides/hooks/receiveBook.ts

```diff
@@ -0,0 +1,22 @@
+import { GetHighlightsSummarySourceTypeEnum } from '@openstax/highlighter/dist/api';
+import { ActionHookBody, AppServices, MiddlewareAPI } from '../../../types';
+import { actionHook } from '../../../utils';
+import { receiveBook } from '../../actions';
+import { receiveStudyGuides } from '../actions';
+
+export const hookBody: ActionHookBody<typeof receiveBook> = (
+  services: MiddlewareAPI & AppServices
+) => async(action) => {
+  if (!services.getState().content.studyGuidesEnabled) { return; }
+
+  const studyGuidesSummary = await services.highlightClient.getHighlightsSummary({
```

Comment:
you're gonna want to use all those highlighting helpers to paginate through these results

Reply:
Should this be done on this pr / for this card?
These helpers need some work to be reusable and I thought I should only apply basic logic for now

Reply:
you're right this doesn't matter because its just the summary, we'll address the utils when we do the modal content